src/app/pages/OriginsLaunchpad/pages/SalesDay/index.module.scss

```diff
@@ -4,10 +4,5 @@
 }
 
 .titleContent {
-  font-size: 2.5rem;
-  font-weight: 600;
-  line-height: 1;
-  letter-spacing: 0;
-  margin-left: 2rem;
-  text-transform: uppercase;
-}
\ No newline at end of file
+  @apply tw-text-4xl tw-font-semibold tw-leading-none tw-tracking-normal tw-ml-8 tw-uppercase;
```

Comment:
In cases like this, you should delete the selector and use the classes directly in the component because you don't add any custom value here.

Reply:
Okay, I understand.